_dashboards/discover/exploring-query-enhancements.md

```diff
@@ -0,0 +1,77 @@
+---
+layout: default
+title: Exploring query enhancements
+parent: Analyzing data
+nav_order: 20
+---
+
+# Exploring query enhancements
+Introduced 2.18
+{: .label .label-purple }
+
+Starting with OpenSearch 2.17, query enhancements have been made. These enhancements are experimental and may be subject to change or instability. Enhancements include the following: 
+
+- Query languages PPL and SQL, with **Query Assist** for PPL
+- Multiline query editor for PPL and SQL and autocomplete for PPL and DQL
+- Data selector with **Data Explorer** that supports index patterns, indexes, and Amazon S3 connections and data configuration that supports selecting the appropriate query language based on the data type
+- Link sharing through URLs without needing write permission to create an index pattern
```

Comment:
creating an index pattern with write permission is required which was a blocker for exploring data. non-privileged users would have to depend on someone with permission to create the index pattern for the data to be explored leading to suboptimal index patterns to be created as it would be easier to just do something like `logs-*` instead of the specific logs. so i think it is right but there's a couple number of features that get unlocked without the required write permissions

Reply:
@kavilla @sejli Should the link sharing bullet be modified to read: Link sharing through URLs without needing write permission to create an index pattern, enabling non-privileged users to access and interact with the data, even if they don't have the ability to define the underlying index patterns

Reply:
LGTM, @kavilla?